script/openqa-bootstrap

```diff
@@ -3,9 +3,9 @@
 
 # add extra repos for leap
 source /etc/os-release
-if $NAME == "openSUSE Leap" ; then
-	zypper -n addrepo obs://devel:openQA devel:openQA
-	zypper -n addrepo obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}
+if [[ "$NAME" == "openSUSE Leap" ]] ; then
```

Comment:
I think proper POSIX shell style would be

```
if [ "$NAME" = "openSUSE Leap" ]; then
```

with the single `[ ` and single `=`. You can check with the tool `checkbashisms`

Reply:
I do this on purpose because it is faster :P

Reply:
I can't counter that ;)

Reply:
The first line of the script is `#!/bin/bash -x` so that's fine. I would rather remove the unnecessary quoting of `$NAME`.

Reply:
I always try to quote string vars in bash because of strange behaviours like this - especially if the string contains spaces:

```
name="foo bar"
# note that this only fails for single [ ]
[ $name = "foo bar" ] ; echo $?
-bash: [: Zu viele Argumente.
2
```

It would actually work for double `[[ ]]` but if I quite strings in bash, I'm on the safe side.

Reply:
ok, what I prefer is to just quote every parameter in a comparison, i.e.

```
#!/bin/sh -e
name="foo bar"
[ "$name" = "foo bar" ]
```

also works with POSIX sh, e.g. dash. Side story: "dash" is part of our distributions as well. I have it installed on my computer as dependency to `gitlab-runner`